setenv.sh

```diff
@@ -1,6 +1,14 @@
 # if running the kdb+tick example, change these to full paths
 # some of the kdb+tick processes will change directory, and these will no longer be valid
-export TORQHOME=${PWD}
+
+if [ "-bash" = $0 ]; then
+    dirpath="${BASH_SOURCE[0]}"
+else
+    dirpath="$0"
+fi
+
+export TORQHOME=$(dirname $dirpath)
```

Comment:
Was this a requested change? Seems somewhat practical but its absent in default TorQ I think.

Reply:
I might need to change it there as well. monit doesn't work if I use `export TORQHOME=${PWD}`